app/_landing_pages/dedicated-cloud-gateways.yaml

```
@@ -0,0 +1,170 @@
+metadata:
+  title: "Dedicated Cloud Gateways"
+  content_type: landing_page
+  description: Learn how Dedicated Cloud Gateways work and how to configure them.
+  tags:
+    - dedicated-cloud-gateways
+    - gateway-manager
+    - data-plane-nodes
+
+rows:
+  - header:
+      type: h1
+      text: "Dedicated Cloud Gateways"
+
+  - columns:
+      - blocks:
+          - type: structured_text
+            config:
+              blocks:
+                - type: text
+                  text: |
+                    Dedicated Cloud Gateways are data plane nodes that are fully managed by Kong in {{site.konnect_short_name}}.
+                    
+                    You don't need to host any data planes, while maintaining control over the size and location of the gateway infrastructure. 
+                    This allows Kong to autoscale your nodes for you and reduces your operational complexity.
+
+                    **Benefits of Dedicated Cloud Gateways:**
+                    - {{site.konnect_short_name}} handles gateway upgrades for you.
+                    - Choose between public and private networking modes.
+                    - Automatic scaling with Autopilot mode.
+                    - Support for AWS and Azure in multiple [regions](/konnect-geos/).
+
+                    You can manage your Dedicated Cloud Gateway nodes in [Gateway Manager](https://cloud.konghq.com/gateway-manager/).
```

```suggestion
                    You can manage your Dedicated Cloud Gateway nodes in [Gateway Manager](https://cloud.konghq.com/gateway-manager/).
             - type: button
            config:
              align: left
              text: Create a Dedicated Cloud Gateway
              url: https://cloud.konghq.com/gateway-manager/create-control-plane
```
The alignment is probably off, but I wanted a card or something where I could create a DCG. I didn't see a way to do this with the API and I don't think we have a how to. And also, I don't think we need a how to since the UI hand holds users when they create one, so I just linked directly to where they can do it in the UI.